docs/02-app/02-api-reference/05-next-config-js/logging.mdx

```diff
@@ -9,12 +9,31 @@ You can configure the logging level and whether the full URL is logged to the co
 
 Currently, `logging` only applies to data fetching using the `fetch` API. It does not yet apply to other logs inside of Next.js.
 
-```js filename="next.config.js"
-module.exports = {
+```js filename="next.config.mjs"
+const nextConfig = {
   logging: {
     fetches: {
       fullUrl: true,
     },
   },
 }
+
+export default nextConfig
+```
+
+In addition, you can disable the route-level logging by setting `logging` to `false`.
```

Comment:
Quick question, should we say disable all logs here?

Reply:
It's better only say disabling nextjs development logging there. cause if there's other error or warning output from console we still let them out